exercises/C-variables/exercise.js

```diff
@@ -1,3 +1,3 @@
 // Start by creating a variable `greeting`
-
+var greeting = "Hello World"
```

Comment:
You should not use var, as this is a legacy keyword. Try to use 'const' if you are not changing the value of the variable, or 'let' if you are changing the content of the variable after the declaration.